src/main/java/com/google/sps/servlets/AddSchoolData.java

```diff
@@ -7,10 +7,12 @@
 import com.google.appengine.api.datastore.FetchOptions;
```

Comment:
consider merging this into your class count PR branch because I think the changes to AddSchoolData.java and AddSchoolData.html are in both PRs